webcompat/static/css/development/components/search-bar.css

```diff
@@ -39,7 +39,12 @@
       font-size: var(--SearchBar-fontSize);
       flex: 1;
       margin: 0 1em;
-      background-color: transparent;
+      background-color: white;
```

Comment:
@magsout something like `--SearchBar-background`? I suppose that could just inherit from `--base-background`?